src/Carbon/Carbon.php

```diff
@@ -88,6 +88,7 @@ class Carbon extends DateTime implements JsonSerializable
     const MONTHS_PER_YEAR = 12;
     const MONTHS_PER_QUARTER = 3;
     const WEEKS_PER_YEAR = 52;
```

Comment:
What about leap years?

Reply:
This constant is 4 years old, and I think anyone complained about it. 52 weeks is 364 days, so it's floor value for both leap years and other years.